[TASK] Document search syntax for suggest wizard (#1062)
TYPO3-Documentation/TYPO3CMS-Reference-TCA: Documentation/ColumnsConfig/Type/Group/Properties/HideSuggest.rst

    The "suggest wizard" is added by default to all database relation group fields. After a couple of characters have
    been typed into this field, an ajax based search starts and shows a list of records matching the search word.

>     The syntax for the search is: `"multiple words" something`, so you can enclose words which need to be
>     found next to another within double quotes.

Reviewer: I miss an `each` here. What do you think about:
```suggestion
    The syntax for the search is: `"multiple words" something`, so you can enclose words you want to find next to each other in double quotes.
```
